x/auth/tx/service_test.go

```diff
@@ -5,13 +5,19 @@ package tx_test
 import (
 	"context"
 	"fmt"
-	"testing"
-
-	"github.com/stretchr/testify/suite"
 
 	"github.com/cosmos/cosmos-sdk/client"
+
+	"strings"
+	"testing"
```

Comment:
could you sort the imports

Reply:
[sorted](https://github.com/cosmos/cosmos-sdk/pull/9038/commits/afe76191c55e224b6503b19093e7e5ba162970ac#diff-61a9cb62b0e38da35032d7bc195c38b2c35c342c7302bf6dde5a483f63a96186L5-L16)